applications/DamApplication/python_scripts/impose_uniform_temperature_process.py

```diff
@@ -36,4 +36,4 @@ def ExecuteInitialize(self):
 
     def ExecuteInitializeSolutionStep(self):
 
-        self.process.ExecuteInitializeSolutionStep()
+        self.process.ExecuteInitialize()
```

Comment:
Is this correct? It makes not so much sense to me to call the ExecuteInitialize() in the ExecuteInitializeSolutionStep() (just wondering...)